client/src/mock_rpc_client_request.rs

```diff
@@ -61,7 +61,7 @@ impl GenericRpcClientRequest for MockRpcClientRequest {
                 }
             }
             RpcRequest::GetBalance => {
-                let n = if self.url == "airdrop" { 0 } else { 50 };
+                let n = if self.url == "airdrop" { 1 } else { 50 };
```

Comment:
@CriesofCarrots mind adding some clarity as to why `0` when `airdrop`? 🙏

Reply:
I believe it was for a mock-airdrop failure test case, which appears to not exist in `test_cli_process_command` at all anymore. (Pretty sure this line is now errant and may as well be removed: https://github.com/solana-labs/solana/blob/268bb1b59b52574660303dd9cf2367a630cd9364/cli/src/cli.rs#L3202)

Reply:
Whew!  I was starting to doubt myself :sweat_smile: 

The mock airdrop has been dropped as suggested :pray: